backend/polygon/polygon_api.py

```diff
@@ -0,0 +1,19 @@
+"""
+Polygon API class to get stock financials and news articles
+https://polygon.io/docs/stocks/
+https://polygon-api-client.readthedocs.io/en/latest/index.html
+"""
+from polygon import RESTClient # type: ignore
+import config #contains API_KEY
```

Comment:
This is a bit clunky. Could we maybe do something like https://djangocentral.com/environment-variables-in-django/?